cmd/kubectl-gadget/profile/cpu.go

```diff
@@ -98,7 +137,9 @@ func runProfileCPU(cmd *cobra.Command, args []string) error {
 			time.Sleep(time.Duration(params.Timeout) * time.Second)
 			c <- os.Interrupt
 		}()
-		fmt.Printf("Capturing stack traces...")
+		if params.OutputMode != utils.OutputModeJSON {
+			fmt.Printf("Capturing stack traces...")
+		}
 	} else {
 		fmt.Printf("Capturing stack traces... Hit Ctrl-C to end.")
```

Comment:
I think there is a missing a conditional on `params.OutputMode != utils.OutputModeJSON` to only print it when no json is used.